guide/arabic/jquery/jquery-html-method/index.md

```diff
@@ -1,8 +1,9 @@
----
-title: HTML Method
-localeTitle: طريقة HTML
----
# طريقة HTML
-
+---
+title: HTML Method
+localeTitle: طريقة HTML
+---
+# طريقة HTML
+jQuery
 تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```

Comment:
Is this an intended change? Doesn't look right here?

Reply:
`^M` what was that?

Reply:
I am not sure, however currently you have:

```
jQuery
تحصل طريقة Jquery `.html()` على محتوى عنصر HTML أو تقوم بتعيين محتوى عنصر HTML.
```

and similar changes in other files.

Reply:
not anymore